toolkit/features/n8n_import.py

```
@@ -0,0 +1,325 @@
+        spec.namespace,
+        "--kubeconfig",
+        kc,
+        "--timeout=120s",
+    ]
+    return _run(status)
+
+
+def _run(cmd: list[str], stdin: str | None = None) -> bool:
+    """Run a kubectl command; return True on success, log stderr on failure."""
+    try:
+        result = subprocess.run(cmd, input=stdin, capture_output=True, text=True, check=True)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)